src/leaderboard.js

```diff
@@ -0,0 +1,91 @@
+export default class LeaderBoard {
```

Comment:
In the [javascript requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) point 4, you are encouraged to use ES6 syntax. In this case the arrow functions are highly recommended for all your functions in the codebase. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a look at this [material](https://stackoverflow.com/questions/42964102/syntax-for-an-async-arrow-function) on how to go about it.